src/api/eventsAPI.js

```diff
@@ -1,708 +1,13 @@
-        "start_date": "2017-07-28T22:00:00.000Z",
-        "loc": {
-          "created_date": "2017-03-21T01:53:56.780Z",
-          "modified_date": "2017-03-21T01:53:56.780Z",
-          "origin_system": "Facebook",
-          "venue": "Montgomery Bell State Park",
-          "locality": "Burns",
-          "region": "TN",
-          "postal_code": "37029",
-          "_id": "58d087b4b7aec90011d75ea0",
-          "location": {
-            "coordinates": [
-              36.099474186875,
-              -87.274139394678
-            ]
-          },
-          "address_lines": [
-            "1020 Jackson Hill Rd"
-          ],
-          "identifiers": [
-            "facebook:133081070551637"
-          ]
-        },
-        "date": "2017-03-21T01:53:56.781Z",
-        "_id": "58d087b4b7aec90011d75ea1",
-        "identifiers": [
-          "facebook:133081070551637"
-        ]
-      }
-    ]
-  }
-}
-/* eslint-enable */
-
-function getEvents() {
-  // return axios.get('https://resistance-calendar.herokuapp.com/v1/events')
-  //   .then(res => res.data);
-
-  return new Promise(resolve => {
-    setTimeout(() => {
-      resolve(temp);
-    }, 500);
-  });
+function getEvents({ page = 0, perPage = 25 }) {
+  return axios.get(`https://resistance-calendar.herokuapp.com/v1/events?page=${page}&per_page=${perPage}`)  // eslint-disable-line max-len
```

Comment:
Maybe we make `https://resistance-calendar.herokuapp.com/v1/` a a const as well.